### PR TITLE
Add details of geomap points feature

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?

- Add details of the new Geomap point layer
- Replace images
- Reformat some of the information for better user experience

### Motivation

[DOCS-5450](https://datadoghq.atlassian.net/browse/DOCS-5450)

### Additional Notes
Do not merge, pending PM review

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5450]: https://datadoghq.atlassian.net/browse/DOCS-5450?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by urseberry** [comment]
Editorial review on https://datadoghq.atlassian.net/browse/DOCS-5514

File: content/en/dashboards/widgets/geomap.md [hunk 1]
```diff
@@ -12,47 +12,76 @@ further_reading:
...
-    * Metric: See the [querying][5] documentation to configure a metric query.
-      * **Note**: The group by tag must include a country ISO Code following the alpha-2 ISO format. You can [generate metrics from ingested logs][6], or manually include the [tags on ingest][4].
-
-2. Optional: configure your view box depending on where you'd like to zoom into on the map by default.
+1. Choose the visualization layer:
+    * **Regions**: Aggregate measures at a country level.
+    * **Points**: Overlay events as points on the map to display geographic event data.
+
+2. Choose the data to graph: <br>
+  **Note**: Support for data sources vary based on the visualization layer you select.
```
Comment (kayayarai) [inline content/en/dashboards/widgets/geomap.md:29]:
```suggestion
  **Note**: Support for data sources varies based on the visualization layer you select.
```

File: content/en/dashboards/widgets/geomap.md [hunk 2]
```diff
@@ -12,47 +12,76 @@ further_reading:
...
+  {{< tabs >}}
+  {{% tab "Regions" %}}
+  |  Data source    | Notes    | 
+  | --------------  | -------- |
+  |Log Events   | The group by tag must include a country ISO Code following the alpha-2 ISO format. You can use the [GeoIP Processor][1] to do this, or manually include the [tags on ingest][2]. See the [Log search documentation][3] to configure a log event query.|
+  |Metric   | The group by tag must include a country ISO Code following the alpha-2 ISO format. You can [generate metrics from ingested logs][4], or manually include the [tags on ingest][2]. See the [querying][5] documentation to configure a metric query.|
```
Comment (kayayarai) [inline content/en/dashboards/widgets/geomap.md:35]:
```suggestion
  |Metric   | The group by tag must include a country ISO Code following the alpha-2 ISO format. You can [generate metrics from ingested logs][4], or manually include the [tags on ingest][2]. See the [querying documentation][5] to configure a metric query.|
```

File: content/en/dashboards/widgets/geomap.md [hunk 3]
```diff
@@ -12,47 +12,76 @@ further_reading:
...
+  |RUM   | See the [RUM documentation][4] to configure a RUM query. |
+
+  [1]: /logs/log_configuration/processors/#geoip-parser
+  [2]: /getting_started/tagging/#defining-tags
+  [3]: /logs/search_syntax/
+  [4]: /real_user_monitoring/explorer/search_syntax/
+  {{% /tab %}}
+  {{< /tabs >}}
+
+3. Optional: configure your view box depending on where you'd like to zoom into on the map by default.
```
Comment (kayayarai) [inline content/en/dashboards/widgets/geomap.md:59]:
```suggestion
3. Optional: Configure your view box to specify where you want to focus the map initially.
```
Something like that?

File: content/en/dashboards/widgets/geomap.md [hunk 4]
```diff
@@ -12,47 +12,76 @@ further_reading:
...
 
 ### Options
 
-#### Global time
-
-On screenboards and notebooks, choose whether your widget has a custom timeframe or uses the global timeframe.
+#### Context links
 
-#### Title
+[Context links][7] are enabled by default, and can be toggled on or off. Context links bridge dashboard widgets with other pages (in Datadog, or third-party).
```
Comment (kayayarai) [inline content/en/dashboards/widgets/geomap.md:65]:
```suggestion
[Context links][7] are enabled by default, you can toggle them on or off. Context links connect dashboard widgets with other pages (in Datadog, or third-party).
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
A few edits, looks like a great update!

**Pull request merged**